malcolm/core/runnabledevicestatemachine.py

```diff
@@ -0,0 +1,41 @@
+from malcolm.core.statemachine import StateMachine
+
+
+class RunnableDeviceStateMachine(StateMachine):
+
+    IDLE = "Idle"
+    CONFIGURING = "Configuring"
+    PRERUN = "PreRun"
+    RUNNING = "Running"
+    POSTRUN = "PostRun"
+    PAUSED = "Paused"
+    REWINDING = "Rewinding"
+    ABORTING = "Aborting"
+    ABORTED = "Aborted"
+
+    def __init__(self, name):
+        super(RunnableDeviceStateMachine, self).__init__(name)
```

Comment:
If we put this at the end, then the superclass can allow transitions from any of our states to Fault or Disabled